gef-ui/templates/includes/application-details/facilities.njk

```diff
@@ -36,7 +36,7 @@
   {% endfor %}
 {% endif %}
 
-{% if facilities.data.length > 0 %}
+{% if facilities.status.code !== 0 %}
```

Comment:
Could this be in the else condition above?

Reply:
Yes good spot :)